recipes/treemix/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+  name: treemix
+  version: 1.12
+
+build:
+  number: 0
+  skip: True # [osx]
+
+source:
+  fn: treemix-1.12.tar.gz
+  url: https://bitbucket.org/nygcresearch/treemix/downloads/treemix-1.12.tar.gz
+  md5: 183f605751a0fa445e249db840907104
+
+requirements:
+  build:
+  - libgcc
+  - gcc ==4.8.5
+  - zlib
+  - gsl >=1.6
+  - boost >=1.42
```

Comment:
Please see the C++ section of the GUIDELINES.md. It points to an example recipe.